content/pages/teams.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Teams",
+  "title": "Enterprise",
   "description": "The power of Tina for the entire team.",
   "headline": "The power of Tina for the entire team",
```

Comment:
```suggestion
  "headline": "The power of Tina for the entire company",
```

Reply:
More company-friendly?

Reply:
There's some more content changes coming from Dan in a follow-up.